src/ui/ui_bundles/ui_bundle.js

```diff
@@ -0,0 +1,86 @@
+import { promisify } from 'bluebird';
+
+const read = promisify(require('fs').readFile);
+const write = promisify(require('fs').writeFile);
+const unlink = promisify(require('fs').unlink);
+const stat = promisify(require('fs').stat);
```

Comment:
Unnecessary `require`s? Could do

```js
import { readFile, writeFile, unlink, stat } from 'fs';

const readFileAsync = promisify(readFile);
// ...
```

Would be great to avoid `require` as much as possible imo.